src/app/templates/index.html

```diff
@@ -53,7 +53,7 @@ <h2>SQLFluff's Fixed SQL</h2>
         <tr>
             <th scope="row">{{violation['code']}}</th>
             <td>{{violation['line_no']}} / {{violation['line_pos']}}</td>
-            <td>{{violation['description']}}</td>
+            <td><pre>{{violation['description'].replace('\\n','\n')}}</pre></td>
```

Comment:
Verrrrry slick! Can you think of a way to get this tested or is this one of those things that's not worth testing in our opinion? I am a maniac for testing

Reply:
Added a test case.

Also added coverage and another test to ensure 100% coverage coz I'm a manic for 100% coverage 😀